pglast/printers/ddl.py

```diff
@@ -1901,8 +2111,8 @@ def drop_stmt(node, output):
             output.print_list(node.objects, ',', standalone_items=False, are_names=True)
     if node.behavior == enums.DropBehavior.DROP_CASCADE:
         output.write(' CASCADE')
-    elif node.behavior == enums.DropBehavior.DROP_RESTRICT:
-        output.write(' RESTRICT')
+    #elif node.behavior == enums.DropBehavior.DROP_RESTRICT:
```

Comment:
I'd prefer a `NotImplementedError` here, instead of commented out code

Reply:
ooh, actually it implemented, remove comments.